templates/navigation/partials/_preview-links.html

```diff
@@ -0,0 +1,17 @@
+<div class="p-navigation__preview-links">
+  {% if preview_secondary_links.title %}
+    <p class="p-navigation--list-heading">{{ preview_secondary_links.title }}</p>
+  {% endif %}
+  <div class="row u-no-padding">
+    {% for link in preview_secondary_links.links %}
+      <div class="col-2">
+        <a class="p-link--inverted" href="">
```

Comment:
The href links aren't being set here for the preview links

Reply:
Good catch 🙈